src/global/constants.js

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
In fact I don't think it is necessary at all do to our change in .eslintrc ignoring the rule for these kind of files. I will fix it.

Reply:
Nice.  I like that we're right-sizing the ESLINT rules.  i.e. adding exceptions that are sensible, so that we can avoid all the suppression comments that end up turning into superfluous noise.  We want the ESLINT rules to be helpful and enforce quality and standardization, but not be an impediment and nag when we're writing perfectly good code.

Reply:
Fixed in db1eed1.  Resolving.